src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto2";
+
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// The team color of the robot
+enum TeamColor {
+    UNKNOWN = 1;
+    YELLOW = 2;
+    BLUE = 3;
+}
+
+message RobotId {
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+}
+
+enum ChipKickDetectionSupport {
+    UNSUPPORTED = 0;
+    FIRST_TOUCHDOWN_ONLY = 1;
+    ALL_TOUCHDOWNS = 2;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 1;
```

Comment:
If this is an indicator for visibility I would name it like that.
On a next level it would actually make sense to add a covariance matrix to have a full picture of uncertainties. But I would consider this optional for this PR.